addons/project/views/project_views.xml

```diff
@@ -603,10 +603,14 @@
                                             <span style="font-weight:bold;" class="fa fa-fw mt4 fa-meh-o text-warning" t-if="record.rating_last_value.value == 5" title="Latest Rating: Not Satisfied" role="img" aria-label="Neutral face"/>
                                             <span style="font-weight:bold;" class="fa fa-fw mt4 fa-frown-o text-danger" t-if="record.rating_last_value.value == 1" title="Latest Rating: Higly Dissatisfied" role="img" aria-label="Sad face"/>
                                         </b>
+                                        <!-- formating of the date -->
+                                        <t name="date_format" t-set="date_format" t-value="'MM/DD/YY'" />
```

Comment:
Don't need the name, you can xpath based on the `t-set`.
No so bad to extract the format in a var though.